routers/api/v1/repo/issue_label.go

```diff
@@ -317,7 +319,32 @@ func prepareForReplaceOrAdd(ctx *context.APIContext, form api.IssueLabelsOption)
 		return nil, nil, err
 	}
 
-	labels, err := issues_model.GetLabelsByIDs(ctx, form.Labels, "id", "repo_id", "org_id", "name", "exclusive")
+	var (
+		labelIDs   []int64
+		labelNames []string
+	)
+	for _, label := range form.Labels {
+		rv := reflect.ValueOf(label)
+		switch rv.Kind() {
+		case reflect.Float64:
```

Comment:
LGTM! Seeing `reflect.Float64` be used here and having the "label IDs" test pass puzzled me for a bit but after reading some of the `jsoniter` code and `encoding/json` [docs](https://pkg.go.dev/encoding/json#Unmarshal) this is expected behavior.